backend/applications/enums.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+
+class ApplicationStatus(models.TextChoices):
+    DRAFT = "draft"
+    ADDITIONAL_INFORMATION_REQUESTED = "additional_information_requested"
+    ADDITIONAL_INFORMATION_PROVIDED = "additional_information_provided"
+    SUBMITTED = "submitted"
+    ACCEPTED = "accepted"
+    REJECTED = "rejected"
+    DELETED_BY_CUSTOMER = "deleted_by_customer"
```

Comment:
As other strings in the project are translatable these could have translatable labels as well.